naplib/naplab/process_ieeg.py

```diff
@@ -692,9 +686,13 @@ def _spectrograms_from_stims(stim_data_dict, stim_order, fs_out, aud_fn, aud_kwa
     specs : list of np.ndarray
         List of same length as stim_order containing the spectrogram for each stimulus
     """
+    if logging.root.level <= logging.INFO:
```

Comment:
if we go with the package level logging this should probably not be logging.root right?